pkgs/by-name/gn/gnucash/package.nix

```diff
@@ -131,6 +131,8 @@ stdenv.mkDerivation rec {
 
     wrapProgram $out/bin/finance-quote-wrapper \
       --prefix PERL5LIB : "${with perlPackages; makeFullPerlPath [ JSONParse FinanceQuote ]}"
+
+    chmod +x $out/share/applications/gnucash.desktop
```

Comment:
Why?

Reply:
So that the Gnome launcher application menu will automatically show it if Gnucash is installed.